Move to material design

vidya-ram opened the pull request:
Baseframe dependency https://github.com/hasgeek/baseframe/pull/263

jace commented:
@vidya-ram Third party JS should be moved into a subfolder. It's hard to tell which bits are ours otherwise.

jace commented on imgee/templates/delete.html.jinja2:
```diff
@@ -6,11 +6,11 @@
+		<p class="mui--text-subhead">Are you sure you want to delete "<strong>{{file.title}}</strong>"?</p>
 		<p id="warning">
-			<i class="icon-warning-sign"></i>&nbsp;<strong>Warning: If you delete it, you will lose all the links wherever this image appears.</strong>
+			{% assets "fa5-sprite" %}<svg class="fa5-icon fa5--align-baseline" aria-hidden="true" role="img"><use xlink:href="{{ ASSET_URL }}#info-circle"></use></svg>{% endassets %}&nbsp;<strong>Warning: If you delete it, you will lose all the links wherever this image appears.</strong>
```
Many templates have mixed tabs and spaces. `pre-commit` and linters are needed here.

jace commented on imgee/templates/index.html.jinja2:
```diff
@@ -1,35 +1,25 @@
 {% extends "layout.html.jinja2" %}
-{% block titletags %}
-  <title>{% block title %}{{ config['SITE_TITLE'] }}{% endblock %}</title>
```
We should drop use of `config['SITE_TITLE']` now as it interferes with i18n. This code is no longer meant for others to run their own installations.

vidya-ram commented:
> @vidya-ram Third party JS should be moved into a subfolder. It's hard to tell which bits are ours otherwise.

I had removed all third party JS to baseframe, missed deleting jquery.jeditable.js. Fixed it now.
